src/js/tabs/osd.js

```diff
@@ -371,6 +365,12 @@ OSD.drawStickOverlayPreview = function() {
         return Math.floor(Math.random() * Math.floor(count));
     }
 
+    const STICK_OVERLAY_SPRITE = [
```

Comment:
While this is an improvement, I can not see how this scope change will fix any bugs. Can you explain please?

Reply:
Moving this is obvious because it's only used in this function. But was not at all what I expected was solving the bug. The array that's build up in this function had undefined symbols. At first I tried to chase charCode but that was only a subsequent behavior. So very strange I found out that the array I moved was not accessible within the function although it was a constant declared outside the block scope.

Reply:
Ok, now I want to know why. Moved the array back but changed the values to use 0x08, 0x09, 0x0A instead to see it that works and indeed in this case the array does not have this problem. So it has something to do with using values from a function in a const?

Reply:
Not sure how this move is relating to `SYM.loadSymbols()`, as it is not connected in any way. But the change of scope makes sense, so good to go.